sources/es/provinces.hjson

```diff
@@ -0,0 +1,71 @@
+{
+    versions: '>=1'
+    type: http
+    production: true
+    note: Spain communities and provinces (some provinces missing, need to be added manually)
```

Comment:
this note is a little ambiguous (?). does this mean the data `has already` been added manually in the geojson, or that they are missing from the geojson and `still` need to be added manually.